bindings/lua/example/fs.lua

```diff
@@ -0,0 +1,28 @@
+-- Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Maybe we need to use:

```
--[[

xxxxx

--]]
```

You can try the following command to auto format it:

```shell
docker run -it --rm -v $(pwd):/github/workspace ghcr.io/korandoru/hawkeye-native format
```